apps/api-reference/src/apis/evm/common.ts

```diff
@@ -0,0 +1,102 @@
+import { z } from "zod";
+
+import {
+  type ReadApi,
+  type WriteApi,
+  type NetworkInfo,
+  EvmApiType,
+  Language,
+} from "../../components/EvmApi";
+import { singletonArray, safeFetch } from "../../zod-utils";
+
+export const readApi = <ParameterName extends string>(
+  spec: Omit<ReadApi<ParameterName>, "children" | "type"> & {
+    description: string;
+  },
+) => ({
+  ...spec,
+  type: EvmApiType.Read,
+});
+
+export const writeApi = <ParameterName extends string>(
+  spec: Omit<WriteApi<ParameterName>, "children" | "type"> & {
+    description: string;
+  },
+) => ({
+  ...spec,
+  type: EvmApiType.Write,
+});
+
+export const BTCUSD =
+  "0xe62df6c8b4a85fe1a67db44dc12de5db330f7ac66b72dc658afedf0f4a415b43";
+export const ETHUSD =
+  "0xff61491a931112ddf1bd8147cd1b641375f79f5825126d665480874634fd0ace";
+
+const HERMES_URL = "https://hermes.pyth.network";
+
+export const getLatestPriceFeed = async (feedId: string) => {
+  const url = new URL("/api/latest_price_feeds", HERMES_URL);
+  url.searchParams.set("ids[]", feedId);
+  url.searchParams.set("target_chain", "evm");
+  url.searchParams.set("binary", "true");
+  return safeFetch(priceFeedSchema, url);
+};
```

Comment:
`/api/latest_price_feeds` is v1 and its deprecated, we should use `/v2/updates/price/latest` instead

Reply:
ah, good to know, will update!

Reply:
Done, thanks!